<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Recent</string>
     </property>
     <addaction name="actionNoRecentFile"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSaveAll"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuCode">
     <property name="title">
      <string>Code</string>
     </property>
     <addaction name="actionComment"/>
     <addaction name="actionUncomment"/>
     <addaction name="actionCommentLines"/>
     <addaction name="actionUncommentLines"/>
     <addaction name="actionIndent"/>
     <addaction name="actionUnindent"/>
    </widget>
    <addaction name="actionSelectAll"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionSearchNext"/>
    <addaction name="actionReplace"/>
    <addaction name="actionGoToLine"/>
    <addaction name="separator"/>
    <addaction name="menuCode"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionResetSettings"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDocumentation"/>
   </widget>
   <widget class="QMenu" name="menuNavigate">
    <property name="title">
     <string>Navigate</string>
    </property>
    <addaction name="actionSend"/>
    <addaction name="actionUnsend"/>
    <addaction name="actionUnsendAll"/>
    <addaction name="actionSendAll"/>
    <addaction name="actionSendToCursor"/>
    <addaction name="separator"/>
    <addaction name="actionInterrupt"/>
    <addaction name="actionRestart"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionOpenedDocuments"/>
    <addaction name="actionFileBrowser"/>
    <addaction name="actionDocumentationPanel"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuNavigate"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAll"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBarCoq">
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionUnsend"/>
   <addaction name="actionSend"/>
   <addaction name="actionUnsendAll"/>
   <addaction name="actionSendAll"/>
   <addaction name="actionSendToCursor"/>
  </widget>
  <widget class="QDockWidget" name="dockFileBrowser">
   <property name="windowTitle">
    <string>File Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockOpenedDocuments">
   <property name="windowTitle">
    <string>Documents</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockDocumentation">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="DocumentationWidget" name="documentationPanel"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Coquille</string>
   </property>
  </action>
  <action name="actionSend">
   <property name="text">
    <string>Send</string>
   </property>
  </action>
  <action name="actionUnsend">
   <property name="text">
    <string>Unsend</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionComment">
   <property name="text">
    <string>Comment Selection</string>
   </property>
  </action>
  <action name="actionUncomment">
   <property name="text">
    <string>Uncomment Selection</string>
   </property>
  </action>
  <action name="actionIndent">
   <property name="text">
    <string>Indent Line</string>
   </property>
  </action>
  <action name="actionUnindent">
   <property name="text">
    <string>Unindent Line</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
   <property name="toolTip">
    <string>Search</string>
   </property>
  </action>
  <action name="actionSearchNext">
   <property name="text">
    <string>Search Next</string>
   </property>
   <property name="toolTip">
    <string>Search</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionCommentLines">
   <property name="text">
    <string>Comment Lines</string>
   </property>
  </action>
  <action name="actionUncommentLines">
   <property name="text">
    <string>Uncomment Lines</string>
   </property>
  </action>
  <action name="actionNoRecentFile">
   <property name="text">
    <string>No recent file</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionDocuments">
   <property name="text">
    <string>Documents</string>
   </property>
  </action>
  <action name="actionFile_Browser">
   <property name="text">
    <string>File Browser</string>
   </property>
  </action>
  <action name="actionShowDocks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Docks</string>
   </property>
  </action>
  <action name="actionOpenedDocuments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Opened Documents</string>
   </property>
  </action>
  <action name="actionUnsendAll">
   <property name="text">
    <string>Unsend All</string>
   </property>
  </action>
  <action name="actionSendAll">
   <property name="text">
    <string>Send All</string>
   </property>
  </action>
  <action name="actionSendToCursor">
   <property name="text">
    <string>Send To Cursor</string>
   </property>
  </action>
  <action name="actionFileBrowser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File Browser</string>
   </property>
  </action>
  <action name="actionInterrupt">
   <property name="text">
    <string>Interrupt</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionGoToLine">
   <property name="text">
    <string>Go To Line</string>
   </property>
  </action>
  <action name="actionResetSettings">
   <property name="text">
    <string>Reset Settings</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionDocumentation_on_this">
   <property name="text">
    <string>Documentation on this</string>
   </property>
  </action>
  <action name="actionDocumentationPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Documentation Panel</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DocumentationWidget</class>
   <extends>QWidget</extends>
   <header>documentationwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
